e2e/test/MethodE2ETests.cs

```diff
@@ -23,6 +24,8 @@ public class MethodE2ETests : IDisposable
         private const string MethodName = "MethodE2ETest";
         private static TestLogging _log = TestLogging.GetInstance();
 
+        private const double defaultMethodTimeoutMinutes = 1;
```

Comment:
Instead of a double, consider making it a static readonly TimeSpan, with value of `TimeSpan.FromMinutes(1)`

Reply:
Good idea